lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Audit a page to ensure that resource loaded over its own
+ * origin are over the http/2 protocol.
```

Comment:
I figure there will be a lot of sites that are using h2 but make http1.1 requests to API endpoints, ad networks, etc. In those cases, there's not much the site author can do.